Gruntfile.js

```diff
@@ -39,13 +39,21 @@ module.exports = function (grunt) {
 		nodeunit: {
 			all: 'test/node.js'
 		},
+		mochaTest: {
```

Comment:
Can we run Babel outside our build system, like inside the tests without having to change the Grunt tasks? See the comment: https://github.com/js-cookie/js-cookie/pull/237#issuecomment-232860139.